lang/cs/Org.Apache.REEF.Tests/Functional/FaultTolerant/PoisonTest.cs

```diff
@@ -87,45 +88,48 @@ public void OnNext(IDriverStarted value)
 
             public void OnNext(IAllocatedEvaluator value)
```

Comment:
Delaying is not a good way to ensure that Task is created because it still ultimately depends on how the CPU schedules your threads. The only way to ensure that is to bind to `TaskStartHandler`. Alternatively, we can change the test to not catch Task failures, but only ensure that the Evaluator has failed.